build-test.cmd

```diff
@@ -265,7 +269,38 @@ if errorlevel 1 (
     exit /b 1
 )
 
+REM Prepare the Test Drop
+REM Cleans any NI from the last run
+powershell "Get-ChildItem -path %__TestWorkingDir% -Include '*.ni.*' -Recurse -Force | Remove-Item -force"
```

Comment:
@ramarag I see - I could put the logic to build Core_Root & sanitize the test build back into runtest, without removing it from build-test. That way we still maintain the advantage of being able to run build-test, then run any individual test case without running runtest first, plus we get a Core_Root built against the most recent product build when we do runtest - but then we have the disadvantage of building Core_Root twice in a row in the CI (build-test & runtest would both build it)